spec/rspec/core/example_spec.rb

```diff
@@ -58,6 +58,21 @@ def metadata_hash(*args)
     end
   end
 
+  it "captures example timing even for exceptions unhandled by RSpec" do
+    begin
+      avoided = RSpec::Support::AllExceptionsExceptOnesWeMustNotRescue::AVOID_RESCUING
+      unhandled = Class.new(StandardError)
+      avoided << unhandled
```

Comment:
Mutating a global constant feels kinda gross.  Can we just use one of the exceptions already in it instead of adding a new one to it?